### PR TITLE
docs: Create read-only-fs.md

### PR DESCRIPTION
**Pull request by rarkins**
## Summary by CodeRabbit

- **New Features**
	- Introduced documentation for support of read-only file systems in version 9.0.0.
- **Documentation**
	- Detailed steps for testing read-only file systems, including configuration for Server and Worker images.
	- Provided guidance on volume mapping and potential risks associated with read-only environments.
	- Suggested enhancements for package managers to mitigate write issues in read-only setups.

### CODE REVIEW

File: docs/read-only-fs.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Support for read-only file systems is currently in testing.
+
+To test it, follow these steps:
+
+## Use pre-release EE images
+
+The custom images https://hub.docker.com/repository/docker/mend/renovate-ee-server and https://hub.docker.com/repository/docker/mend/renovate-ee-worker have been built to support read-only FS.
+Use whatever is the latest `9.0.0-pre.x` tag available.
+
+The `-full` worker image is now yet supported.
```
Comment (justo-mend) [inline docs/read-only-fs.md:12]:
```suggestion
The `-full` worker image is not yet supported.
```

File: docs/read-only-fs.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Map read-write volumes
+
+Ensure that the EE Server has a read-write `/tmp` volume.
+
+Ensure that the EE Worker has read-write `/tmp` and `/opt/containerbase` volumes.
+
+## Other volumes
+
+The main "risk" of a read-only FS for Renovate is that there are dozens of package managers which can be called, and those package managers can choose to write files into unexpected locations.
```
Comment (justo-mend) [inline docs/read-only-fs.md:26]:
```suggestion
The main "risk" of a read-only FS for Renovate is that there are dozens of package managers that can be called, and those package managers can choose to write files into unexpected locations.
```

**Review by justo-mend — CHANGES_REQUESTED** [review]
Two tweaks to wording

**Review by justo-mend — DISMISSED** [review]
LGTM

**Comment by justo-mend** [comment]
LGTM

File: docs/read-only-fs.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Support for read-only file systems is available from version 9.0.0
+
+To test it, follow these steps:
+
+## Use pre-release EE images
+
+The official release images:
+* Community: `ghcr.io/mend/renovate-ce:9.0.0`
+* Enterprise: `ghcr.io/mend/renovate-ee-server:9.0.0` and `ghcr.io/mend/renovate-ee-worker:9.0.0`
+
```
Comment (nabeelsaabna) [inline docs/read-only-fs.md:12]:
```suggestion

## Use the official release images:
* Community: `ghcr.io/mend/renovate-ce:9.0.0`
* Enterprise: `ghcr.io/mend/renovate-ee-server:9.0.0` and `ghcr.io/mend/renovate-ee-worker:9.0.0`

```

File: docs/read-only-fs.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Support for read-only file systems is available from version 9.0.0
+
+To test it, follow these steps:
+
+## Use pre-release EE images
+
+The official release images:
+* Community: `ghcr.io/mend/renovate-ce:9.0.0`
+* Enterprise: `ghcr.io/mend/renovate-ee-server:9.0.0` and `ghcr.io/mend/renovate-ee-worker:9.0.0`
```
[the remark this answers is not in the document]
Comment (justo-mend) [reply docs/read-only-fs.md]: Fixed

**Pull request merged**